packages/plugins/auth0/src/index.ts

```diff
@@ -9,7 +9,7 @@ const { decode, verify } = jwtPkg;
 
 export type Auth0PluginOptions = {
   domain: string;
-  audience: string;
+  audience: string | RegExp | Array<string | RegExp> | undefined;
```

Comment:
Is it possible to do something like this ?
```suggestion
  audience: VerifyOptions['audience'];
```

Reply:
It was indeed possible and a better way to do it. New commit pushed with the requested change.